mappings/net/minecraft/world/loot/LootManager.mapping

```diff
@@ -8,4 +8,7 @@ CLASS cog net/minecraft/world/loot/LootManager
 	METHOD a getSupplierNames ()Ljava/util/Set;
 	METHOD a toJson (Lcoe;)Lcom/google/gson/JsonElement;
 	METHOD a check (Lcof;Lqs;Lcoe;Ljava/util/function/Function;)V
+		ARG 1 arg2
+		ARG 2 supplier
+		ARG 3 getSupplier
```

Comment:
`getSupplier` as an argument name seems odd. what is it for?

Reply:
It's a functional interface of type `Function<Identifier, LootSupplier>` which in the code is hooked to `this.suppliers::get`, a get method of a map.
So a function that takes an identifier and returns the corresponding lootSupplier.
Maybe `getSupplierFunction` would be more clear?

Reply:
getterSupplier?

Reply:
`supplierGetter`?